pkg/identity/certificated.go

```diff
@@ -224,31 +199,60 @@ func Certificated(idConfig *config.IdentityConfig, stopChan <-chan struct{}) (er
 			if err != nil {
 				log.Errorf("Failed to retrieve x509 certificate from identity provider: %s", err.Error())
 			}
+		} else if idConfig.KeyFile != "" && idConfig.CertFile != "" {
+			log.Debugf("Attempting to load x509 certificate from local file: key[%s], cert[%s]...", idConfig.KeyFile, idConfig.CertFile)
+
+			fileBackupCertPEM, err := ioutil.ReadFile(idConfig.CertFile)
+			if err != nil {
+				log.Warnf("Error while reading x509 certificate from local file[%s]: %s", idConfig.CertFile, err.Error())
+			}
+			fileBackupKeyPEM, err := ioutil.ReadFile(idConfig.KeyFile)
+			if err != nil {
+				log.Warnf("Error while reading x509 certificate key from local file[%s]: %s", idConfig.KeyFile, err.Error())
+			}
+
+			fileBackupIdentity, err := InstanceIdentityFromPEMBytes(fileBackupCertPEM)
+			if err != nil {
+				log.Warnf("Error while parsing x509 certificate from local file: %s", err.Error())
+			}
+
+			if fileBackupIdentity == nil || len(fileBackupKeyPEM) == 0 {
```

Comment:
@mlajkim  Are these better to be called as `fileIdentity` and `fileKeyPEM` or are they still better to be called as `fileBackupIdentity` and `fileBackupKeyPEM`?

Reply:
@mlajkim Sounds good with `localFileIdentity` & `localFileKeyPEM`.

Reply:
Handled in https://github.com/AthenZ/k8s-athenz-sia/pull/53/commits/017fc2779fd9d5c8a41032cb9f6ab6e27b39369d